lib/meta.nix

```diff
@@ -162,5 +162,12 @@ rec {
        getExe' pkgs.imagemagick "convert"
        => "/nix/store/5rs48jamq7k6sal98ymj9l4k2bnwq515-imagemagick-7.1.1-15/bin/convert"
   */
-  getExe' = x: y: "${lib.getBin x}/bin/${y}";
+  getExe' = x: y:
+    if ! lib.isDerivation x
+    then throw "lib.meta.getExe': The first argument is of type ${lib.typeOf x}, but it should be a derivation instead."
```

Comment:
`lib.typeOf` doesn't exist, it's `builtins.typeOf`.

It looks like `getExe`/`getExe'` aren't covered by any `lib` tests. Could you add some to `lib/tests/misc.nix`?

Reply:
> `lib.typeOf` doesn't exist, it's `builtins.typeOf`.

oops

Reply:
I'm a bit lost as to how I'm supposed to test this... because there's no `pkgs.` in scope in that file and i need a derivation to check if it works.